components/automate-ui/Makefile

```diff
@@ -60,12 +62,25 @@ license_scout:
 	cd /tmp/license_scout && bundle install --without=development --quiet && echo "license_scout version" && git rev-parse HEAD
 	/tmp/license_scout/bin/license_scout
 
+# These are shortcuts for generating new components in various subdirs.
```

Comment:
PR bonus: Added a new convenience helper in the Makefile--and gave a concrete example so folks will know how to use these! 😁